### PR TITLE
Updated the coc

### PR DESCRIPTION
**Pull request by Sloth-Panda**
correponds to issue -> #5

### CODE REVIEW

File: Code_of_Conduct.md
Comment (praveenscience) [inline Code_of_Conduct.md:48]: Single Lines Please...

File: Code_of_Conduct.md
Comment (praveenscience) [inline Code_of_Conduct.md:10]: Remove 1 Empty Line.

File: Code_of_Conduct.md
Comment (praveenscience) [inline Code_of_Conduct.md:26]: Remove 1 Empty Line.

File: Code_of_Conduct.md
Comment (praveenscience) [inline Code_of_Conduct.md:5]: This one's still not rephrased. It's alright, but which parts have you rephrased, can you please tell?

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please fix these and let's merge this...

File: Code_of_Conduct.md
Comment (Sloth-Panda) [reply Code_of_Conduct.md:48]: you want me to remove the paragraph? or keep it as it and make everything single line

File: Code_of_Conduct.md
Comment (Sloth-Panda) [reply Code_of_Conduct.md:26]: ok

File: Code_of_Conduct.md
Comment (Sloth-Panda) [reply Code_of_Conduct.md:5]: I have rephrased all the parts except this part as it contains specific words as you can see so I just changes 2/3 words in this section only

**Review by praveenscience — APPROVED** [review]
Looks good to me! 👍

File: Code_of_Conduct.md
Comment (praveenscience) [reply Code_of_Conduct.md]: You did it right.

**Comment by praveenscience** [comment]
Mentors, please check. When merging make sure you specify the Fixes #5 on this.

**Comment by Sloth-Panda** [comment]
@praveenscience  sir can you also add swoc to it

**Comment by Sloth-Panda** [comment]
@praveenscience  can you please check this one I see, you assigned me again on the same issue but it's pr is here

**Comment by praveenscience** [comment]
@Sloth-Panda Yea, for some reason, I told you're assigned but really didn't "assign" you the issue. Now I assigned it and as you can see, it's great. Just waiting for one more review from either of the mentors. Reach out to them and get your approval and points bro! 😁

**Comment by Sloth-Panda** [comment]
@Salonii02  and @manudd1011 can you guys please review this

**Review by Salonii02 — APPROVED** [review]

**Comment by Sloth-Panda** [comment]
@praveenscience  does it require 3 members for approval?

**Comment by praveenscience** [comment]
@Sloth-Panda 2 of them. I am extremely busy, so don't worry, I am merging it now. Have patience, we're on different time zones and I am in a full time job. 😊

**Comment by praveenscience** [comment]
@Sloth-Panda Did you see the error message what it says? 🤔

**Comment by Sloth-Panda** [comment]
No , I don't see any error messages. Where do you see though?

**Comment by praveenscience** [comment]
> does it require 3 members for approval?

Why did you ask this @Sloth-Panda?

**Comment by Sloth-Panda** [comment]
As I was trying to reachout to maintainers so wanted to know how many maintainers should I reach out to for reviewing this pr as it was pending for nearly 10 days @praveenscience

**Comment by praveenscience** [comment]
@Sloth-Panda Just one man. It says there, 2 approvals are required. One from me and one from one of the mentors.

**Pull request merged**
